pkg/specgen/generate/container.go

```diff
@@ -540,3 +541,41 @@ func mapSecurityConfig(c *libpod.ContainerConfig, s *specgen.SpecGenerator) {
 	s.Groups = c.Groups
 	s.HostUsers = c.HostUsers
 }
+
+// Check name looks for existing containers/pods with the same name, and modifies the given string until a new name is found
```

Comment:
If you force users to name their pods/containers, then none of this is necessary.

Reply:
Can you rename `kind` to `isCtr` or something similar?

Reply:
Still open.  It would be helpful to make the comment of the function more verbose such that new eyes won't have to read the function.